hack/builder/Dockerfile

```diff
@@ -67,9 +67,11 @@ ENV PATH=$PATH:/opt/gradle/gradle-${GRADLE_VERSION}/bin \
     JAVA_HOME=/usr/lib/jvm/java-11
 
 # reference to master is for an external repo and can't yet be changed
-RUN mkdir -p /gimme && curl -sL \
-    https://raw.githubusercontent.com/travis-ci/gimme/master/gimme | \
-    HOME=/gimme bash >> /etc/profile.d/gimme.sh
+RUN mkdir -p /gimme && curl -L \
```

Comment:
Is it enough to just not run curl in silent mode? I am not sure if the rest of the changes are required?

Reply:
I think is not a problem just like bazel download use:
``` 
RUN curl -L -o /usr/bin/bazel https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-linux-${BAZEL_ARCH} && \
    chmod u+x /usr/bin/bazel
```